pkg/openshift/rosa/oidcconfig.go

```diff
@@ -45,11 +46,16 @@ func (r *Provider) createOIDCConfig(ctx context.Context, prefix, installerRoleAr
 		"--output", "json",
 		"--mode", "auto",
 		"--prefix", prefix,
-		"--managed=false",
 		"--installer-role-arn", installerRoleArn,
 		"--yes",
 	}
 
+	// FedRamp Workaround the oidc needs to be RH Managed
```

Comment:
```suggestion
	// The OIDC needs to be `--managed` for FedRamp
```
Also, instead of adding the `managed` param, if we do put a `fedRamp` field in the provider, you can reference that instead of the `managed` variable